### PR TITLE
Use the boto3 copy method.

### PR DESCRIPTION
**Pull request by MancunianSam**
From the AWS docs:
This is a managed transfer which will perform a multipart copy in multiple threads if necessary

I've removed the test which tests for a standard transfer and we've had
to stop counting the parts as it's not consistent.

### CODE REVIEW

File: python/lambdas/copy-files-from-tdr/lambda_function_test.py [hunk 1]
```diff
@@ -8,43 +8,16 @@
...
-        expected_sqs_args = \
-            {'MessageBody': '{"location": "s3://destination-bucket/test-file"}', 'QueueUrl': 'destination-queue'}
-
-        self.assertEqual(mock_copy_object.call_count, 2)
-        assert_copy_object_arguments(0, 'test-file')
-        assert_copy_object_arguments(1, 'test-file.metadata')
-        self.assertEqual(mock_send_message.call_args_list[0][1], expected_sqs_args)
-
     @patch('lambda_function.s3_client.head_object')
     @patch('lambda_function.s3_client.create_multipart_upload')
```
Comment (techncl) [inline python/lambdas/copy-files-from-tdr/lambda_function_test.py:12]: Is the reason why you have not removed these is because boto3 calls these?

**Review by techncl — COMMENTED** [review]
Looks good! Just some minor comments

File: python/lambdas/copy-files-from-tdr/lambda_function_test.py (hunk 1 above)
Comment (MancunianSam) [reply python/lambdas/copy-files-from-tdr/lambda_function_test.py:11]: I think it does yeah. I did remove them and it stopped working so I think so.

File: python/lambdas/copy-files-from-tdr/lambda_function.py [hunk 2]
```diff
@@ -15,68 +15,14 @@ def lambda_handler(event, context):
...
-            MultipartUpload={'Parts': parts}
-        )
-        print(f"Multipart copy of {s3_key} completed successfully.")
+        s3_client.copy(copy_source, destination_bucket, s3_key)
+        return f"s3://{destination_bucket}/{s3_key}"
     except Exception as e:
-        print(f"Error during multipart copy of {s3_key}: {e}")
-        # If there is an error, abort the multipart upload
-        s3_client.abort_multipart_upload(Bucket=destination_bucket, Key=s3_key, UploadId=upload_id)
+        print(f"Error during copy of {s3_key}: {e}")
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply python/lambdas/copy-files-from-tdr/lambda_function.py:28]: Good idea, I've added it in.

**Comment by MancunianSam** [comment]
Sorry, I forgot the other PR you approved was set to merge to this branch so this has got the fileId changes you've already looked at in it.

**Review by techncl — APPROVED** [review]
👍

**Pull request merged**